spec/models/duration_attribute_accessor_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe NxtSupport::DurationAttributeAccessor do
+  let!(:db_schema) do
+    ActiveRecord::Schema.define do
+      self.verbose = false
+
+      create_table :courses, force: true do |t|
+        t.string :class_duration
+        t.string :topic_duration
+        t.string :total_duration
+      end
+    end
+  end
+
+  describe 'A class with assignable_values' do
+    subject do
+      Class.new(ActiveRecord::Base) do
+        self.table_name = 'courses'
+        include NxtSupport::DurationAttributeAccessor
+
+        duration_attribute_accessor :class_duration, :topic_duration, :total_duration
+      end
+    end
+
+    context 'when attributes have ActiveSupport::Duration type' do
```

Comment:
I think it would also be good to test the cases of assigning `nil` or an invalid type.